homeassistant/components/deconz/const.py

```diff
@@ -1,8 +1,4 @@
 """Constants for the deCONZ component."""
-import logging
-
-_LOGGER = logging.getLogger(".")
```

Comment:
You can replace `"."` with`__package__`

Reply:
OK, then I revert my modifications regarding logging.